stratum/hal/lib/common/admin_service_test.cc

```diff
@@ -166,7 +166,7 @@ TEST_P(AdminServiceTest, CancelReboot) {
   ::gnoi::system::RebootResponse resp;
   ASSERT_OK(admin_service_->Setup(false));
 
-  req.set_delay(5000000);  // 5ms
+  req.set_delay(10000000);  // 5ms
```

Comment:
This belongs to another patch, please remove this commit.
Also, need to update the comment

Reply:
I've removed it and refined the test cases.